release_docs/RELEASE.txt

```diff
@@ -530,6 +530,14 @@ Bug Fixes since HDF5-2.0.0 release
 
     Configuration
     -------------
+    - Use pre-installed libaec compression library
+
+      The CMake logic for finding the libaec compression library has been
+      modified for a system-installed version of the library. Two options
+      must be set, HDF5_ALLOW_EXTERNAL_SUPPORT:STRING=NO and *_USE_EXTERNAL:BOOL=OFF.
```

Comment:
Is there is only one *_USE_EXTERNAL:BOOL=OFF option to set?  Is it libaec_USE_EXTERNAL:BOOL=OFF, or one or more others?